scene/gui/tabs.cpp

```diff
@@ -593,6 +593,58 @@ void Tabs::remove_tab(int p_idx) {
 	_ensure_no_over_offset();
 }
 
+Variant Tabs::get_drag_data(const Point2 &p_point) {
+
+	int hover_now = -1;
+	for (int i = 0; i < tabs.size(); i++) {
+
+		if (i < offset)
+			continue;
+
+		Rect2 rect = get_tab_rect(i);
+		if (rect.has_point(p_point)) {
+			hover_now = i;
+		}
+	}
+
+	return hover_now;
+}
+
+bool Tabs::can_drop_data(const Point2 &p_point, const Variant &p_data) const {
+
+	int hover_now = -1;
+	for (int i = 0; i < tabs.size(); i++) {
+
+		if (i < offset)
+			continue;
+
+		Rect2 rect = get_tab_rect(i);
+		if (rect.has_point(p_point)) {
+			hover_now = i;
+		}
+	}
+
+	return (hover_now > -1);
```

Comment:
Still thinking it might be good idea but on second thought not 100% sure. In my own project I would prefer to have new function with nice self explanatory name that could be used in all three places.  So my original question is still important for me (especially for the future). Will leave for now like that, I'm curious for other opinions :)